code/src/reducers/todos.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const todos = createSlice({
+  name: 'todos',
+  initialState: {
+    items: []
+  },
+  reducers: {
+    addTodo: (state, action) => {
+      const { newItem } = action.payload
+      state.items.push(newItem)
+    },
+    setComplete: (state, action) => {
+      const { itemIndex, completed } = action.payload
+      state.items[itemIndex].completed = completed
+    },
+    removeTodo: (state, action) => {
+      const { itemIndex } = action.payload
+      state.items.splice(itemIndex, 1)
+    },
+    removeAll: (state) => {
+      state.items = []
+    }
+  }
+})
```

Comment:
I like how clean this is! I saw that Jennie posted a link in your slack-group about adding an id to your todos, that's a good idea to be able to control your todo's better since the id is unique and the index can change.